include/bouncer.h

```diff
@@ -379,6 +382,14 @@ struct PgSocket {
 #define tmp_sk_linkfd	query_start
 /* takeover_clean_socket() needs to clean those up */
 
+struct SocketPriority {
+	struct AANode tree_node;	/* used to attach user to tree */
+	char *name;		/* The application_name to be assigned priority. */
+	uint16_t priority;	/* The priority value of sockets matching name. */
+};
+
+#define DEFAULT_SOCKET_PRIORITY 1
```

Comment:
Can we make the default priority higher so that there is room under it for prioritizes which are lower than the default?